my_game.py

```diff
@@ -155,13 +155,18 @@ def on_update(self, delta_time):
 
 class Asteroid(arcade.Sprite):
 
-    def __init__(self):
+    valid_sizes = {1: "images/Meteors/meteorGrey_tiny1.png", 2: "images/Meteors/meteorGrey_small1.png", 3: "images/Meteors/meteorGrey_med1.png", 4: "images/Meteors/meteorGrey_big1.png"}
 
-        # Initialize the asteroid
+    def __init__(self, size=None):
 
+        # Initialize the asteroid
+        
+        if size is None:
+            size = random.randrange(1, 4)
```

Comment:
Let's get the _1_ and _4_ from the keys in _valid_sizes_. Maybe a random choice from a list of valid sizes would be better?

Reply:
I renamed valid_sizes to graphics, and i made a tuple called valid_sizes for picking the size, because that made more sense to me.